soh/soh/Enhancements/randomizer/randomizer_check_tracker.cpp

```diff
@@ -977,11 +1014,17 @@ void CheckTrackerWindow::DrawElement() {
                 doAreaScroll = false;
             }
             for (auto rco : objs) {
-                if (doDraw && isThisAreaSpoiled && IsVisibleInCheckTracker(rco))
-                    DrawLocation(rco);
+                if (IsVisibleInCheckTracker(rco)) {
+                    if (doDraw && isThisAreaSpoiled) {
+                        DrawLocation(rco);
+                    }
+                } else {
+                    
+                }
```

Comment:
is there a reason to have this split up like this with the empty else instead of just having

```suggestion
                if (IsVisibleInCheckTracker(rco) && doDraw && isThisAreaSpoiled)
                    DrawLocation(rco);
                }
```

Reply:
Oh, lol, I think that was leftover from an attempt to get it working a different way. I'll check to make sure I didn't miss implementing something and remove it if not.